### ISSUE TITLE
Lost alll the video resumes

### ISSUE THREAD

**Issue by gerroon** [jellyfin/jellyfin#66]
Hi

I transitioned from Emby to Jellyfin on Debian. Almost everything works, except that I lost all my video positions, all the played videos with resumes now look like they were never played, or started at all.

I do not remember my EMby version unfortunately.

**Comment by DraZtiK** [comment]
Have you tried to copy over your data from emby to jellyfin?

**Comment by gerroon** [comment]
Naturally that is what I did. I am using the Emby database

**Comment by drakus72** [comment]
> Naturally that is what I did. I am using the Emby database

Depending on the version of Emby that you used previously, the database is probably not compatible with jellyfin. In Emby 3.6 they change a lot of the database structure from my understanding and it was not able to be reversable back to the old database structure.

**Comment by JustAMan** [comment]
I don't think that creation of down-migration from Emby 3.6 to Jellyfin would have high priority now... unless there's demand for that? :) I mean, that's another feature which must be prioritized with others, and I do not know whether it is important or not (that is, my db for now has two series and one movie xD as I lost my previous DB a couple of years ago and now am starting from scratch).

**Comment by joshuaboniface** [comment]
Yea I don't think we should support down-migration. It's just a massive hassle for us with little benefit in the long-term. Worse-case we require rebuilding libraries after downgrading and leave it at that. It's unfortunate but one of the side effects of a hard fork like this.

**Comment by drakus72** [comment]
Down-migration would be too much of a head ache, catching up to Emby 3.6 database would be much easier than trying to downgrade.

If you want a true drop in, this would really be the only option unless you did your own database improvements and converted the emby database structure to the jellyfin database structure.

Just an option and my opinion.

**Comment by gerroon** [comment]
Well if you want people to adapt Jellyfin, this database conversion should a priority. If you have been using Emby for years and if you knew you will loose all those settings? Would regular Trump move to Jellfyfin in such case?

**Comment by drakus72** [comment]
I will get a sample database from both and get you some links to them.

**Comment by JustAMan** [comment]
When making a sample db please try to cover as much different cases as you can - movies, music, whatever, resume positions, etc. Thanks!

**Comment by drakus72** [comment]
Going to add at least 1 show 1 movie 1 to music, etc, otherwise it will grow fast to over 100m.

**Comment by sparky8251** [comment]
Any word on whether or not we are going to take the time and effort to backport db changes? If not, maybe this should be closed since its been 30 days since anything has happened here.

**Comment by JustAMan** [comment]
> Official answer: rebuild library under Jellyfin.

Addition to official answer: one can create a new issue requesting a migration script, but without DB examples of "emby 3.6", "emby 3.5" and "jf 10.0.0" if won't have core attention.
